changelog.lua

```diff
@@ -0,0 +1,374 @@
+local pl_template = require "pl.template"
+local pl_tablex = require "pl.tablex"
+local pl_file = require "pl.file"
+local pl_dir = require "pl.dir"
+local pl_path = require "pl.path"
+local pl_stringx = require "pl.stringx"
+local lyaml = require "lyaml"
+local pl_app = require 'pl.lapp'
+local cjson = require "cjson"
+local http = require("socket.http")
```

Comment:
Please do not use `lua-http`, this library had significant quality issues and we had to remove it from our codebase completely.